### ISSUE TITLE
Can't download from abc.com (unable to extract post url)

### ISSUE THREAD

**Issue by dano-code** [yt-dlp/yt-dlp#2952]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.02.04**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

United States

### Description

Can't download from abc.com. yt-dlp has been working fine lately for this and other abc.com tv shows, but started failing on all of them this morning.

### Verbose log

```shell
[debug] Command-line config: ['-vU', '--ap-mso=Comcast_SSO', '--ap-username=PRIVATE', '--ap-password=PRIVATE', '--cookies-from-browser', 'edge', 'https://abc.com/shows/general-hospital/episode-guide/2022-03/03-general-hospital-30322', '-o', 'General Hospital.S59E232.30322.mp4']
[Cookies] Extracting cookies from edge
[debug] Extracting cookies from: "C:\Users\dan\AppData\Local\Microsoft\Edge\User Data\Default\Network\Cookies"
[Cookies] Extracted 4464 cookies from edge
[debug] cookie version breakdown: {'v10': 4464, 'other': 0, 'unencrypted': 0}
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, err utf-8, pref cp1252
[debug] yt-dlp version 2022.02.04 [c1653e9] (win_exe)
[debug] Python version 3.8.10 (CPython 64bit) - Windows-10-10.0.19044-SP0
[debug] exe versions: ffmpeg 4.2.2, ffprobe 4.2.2
[debug] Optional libraries: Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
Latest version: 2022.02.04, Current version: 2022.02.04
yt-dlp is up to date (2022.02.04)
[debug] [Go] Extracting URL: https://abc.com/shows/general-hospital/episode-guide/2022-03/03-general-hospital-30322
[Go] 03-general-hospital-30322: Downloading webpage
[Go] VDKA26617393: Downloading JSON metadata
[debug] Loading ap-mvpd.ABC from cache
[Go] VDKA26617393: Downloading Provider Redirect Page
ERROR: [Go] Unable to extract post url; please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using -U; please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using -U
  File "yt_dlp\extractor\common.py", line 612, in extract
  File "yt_dlp\extractor\go.py", line 238, in _real_extract
  File "yt_dlp\extractor\adobepass.py", line 1484, in _extract_mvpd_auth
  File "yt_dlp\extractor\adobepass.py", line 1391, in post_form
  File "yt_dlp\extractor\common.py", line 1195, in _html_search_regex
  File "yt_dlp\extractor\common.py", line 1186, in _search_regex
```

**Comment by jjs6725** [comment]
I have been trying to download from ABC as well today and I am getting the same exact error and stack trace. I'm also an Xfinity subscriber and my command line config is exactly the same.

**Comment by pukkandan** [comment]
Comcast support is broken https://github.com/yt-dlp/yt-dlp/issues/516

**Comment by jjs6725** [comment]
This isn't just a Comcast issue though, I get this error no matter what TV provider I try to use. Like the OP said, it was working a few days ago until ABC changed something on their site. The real issue is: "Unable to extract post url", I can't even use cookies anymore to authorize it because it can't find the provider page.

**Comment by jiangjiangflies** [comment]
Hi, I have the same issue with abc.com. Though this issue is closed, I can't find any answers relevant to the issues of abc.com.  Is it going to be fixed? I can provide an account if you need one to fix this issue.

**Comment by jjs6725** [comment]
According to issue 516 that pukkandan linked to, it looks like it is really Comcast Xfinity support that is broken. Downloading shows from abc.com that do not require a TV subscriber login works fine for me. Something with Xfinity's login system is breaking yt-dlp's attempts to authenticate with the website. If you know someone who has a TV cable service other than Xfinity, like Verizon, Spectrum, AT&T, Dish, DirecTV, and others, you can try using their login. I'm really sorry, until the issue gets fixed, that is the only thing I can suggest.

**Comment by jiangjiangflies** [comment]
I tried DirecTV, didn't work neither.

**Comment by pukkandan** [comment]
DirecTV is not supported https://github.com/yt-dlp/yt-dlp/issues/1953

**Comment by jiangjiangflies** [comment]
Then which TV subscription is now supported to login in abc.com?

**Comment by shirt-dev** [comment]
> DirecTV is not supported #1953

DirecTV is supported, it's different from DirecTV stream

**Comment by jriker1** [comment]
This is odd.  Posts above are from 2022 but mine stopped working two weeks ago or so.  Doesn't look like this was ever solved but closed.

**Comment by bashonly** [comment]
@jriker1 it was closed as a duplicate of #516, which is still open
